perception/probabilistic_occupancy_grid_map/config/pointcloud_based_occupancy_grid_map.param.yaml

```diff
@@ -18,4 +18,10 @@
     # center of the grid map
     gridmap_origin_frame: "base_link"
     # ray-tracing center: main sensor frame is preferable like: "velodyne_top"
-    scan_origin_frame: "base_link"
+    scan_origin_frame: "velodyne_top"
```

Comment:
The frame velodyne top is not defined as an official AWF tf, so could you please make it a base_link?

Reply:
I see, then I will revert this flag to false and scan_origin_frame to "base_link", and only use "velodyne_top" link name for internal use only.